frontend/public/components/utils/dropdown.jsx

```diff
@@ -118,21 +118,21 @@ class DropDownRow extends React.PureComponent {
 
     let prefix;
     if (onUnBookmark) {
-      prefix = <a className={classNames('bookmarker', {focus: selected, hover})} onClick={e => onUnBookmark(e, itemKey)}><i className="fa fa-minus-circle" /></a>;
+      prefix = <a className={classNames('bookmarker', {hover, focus: selected})} onClick={e => onUnBookmark(e, itemKey)}><i className="fa fa-minus-circle" /></a>;
```

Comment:
These links should have an `href` also. The icons should have `aria-hidden="true"`